sentinel-core/src/main/java/com/alibaba/csp/sentinel/SphU.java

```diff
@@ -242,4 +242,31 @@ public static AsyncEntry asyncEntry(String name, EntryType type) throws BlockExc
     public static AsyncEntry asyncEntry(String name, EntryType type, int count, Object... args) throws BlockException {
         return Env.sph.asyncEntry(name, type, count, args);
     }
+
+    /**
+     * Checking all {@link Rule}s related the resource. The entry may have priority.
+     *
+     * @param name        the unique name for the protected resource
+     * @param prioritized whether the entry is prioritized
+     * @throws BlockException if the block criteria is met, eg. when any rule's threshold is exceeded.
+     * @since 1.4.0
+     */
+    public static Entry entryWithPriority(String name, boolean prioritized) throws BlockException {
```

Comment:
The `prioritized` parameter can be removed, since the method name `entryWithPriority` already indicates the priority meaning.